### PR TITLE
Update offa.csl

### PR DESCRIPTION
**Pull request by nmueller18**
Fixed two small quirks concerning the position of the edition number and the volume of a series.

### CODE REVIEW

File: offa.csl [hunk 1]
```diff
@@ -210,12 +210,14 @@
...
-        <choose>
-          <if match="none" variable="edition">
-            <text value=""/>
-          </if>
-        </choose>
-        <group delimiter="" prefix=" (" suffix=")">
+        <group delimiter="" prefix=" ">
+          <choose>
+            <if match="none" variable="edition">
+              <text macro="edition"/>
```
Comment (adam3smith) [inline offa.csl:216]: same here -- what's the goal of this?

Comment (adam3smith) [reply offa.csl:216]: I'm talking about l. 215-216, which are from your change (in both styles): you're first checking that edition _isn't_ present, then you're printing the edition macro, which does nothing but render the edition.

Comment (nmueller18) [reply offa.csl:216]: Many thanks for the clarification and sorry for the hassle! As I only deal with CSL every few months, I do not really know what I am doing, to be honest. Now I see that this declaration is non-sensical. I suppose
```		
<macro name="place-date">
  <group delimiter="" prefix=" ">
    <choose>
      <if type="book thesis chapter paper-conference" match="any">
         <text macro="edition"/>
           <group delimiter="" prefix="(" suffix=")">
              .
              .
              .
       </group>
      </if>
    </choose>
   </group>
  </macro>
```
would make more sense?

Comment (adam3smith) [reply offa.csl:216]: You might actually want a space for the delimiter, all the way at the top, so you get "1st edition (New York)" and not "1st editions(New York)" but you'd want to test. 
But in terms of logic, yes, this is what you'd want.

**Pull request merged**